xc/common/cmake/install.cmake

```diff
@@ -56,20 +56,20 @@ function(DEFINE_XC_TOOLCHAIN_TARGET)
             ${symbiflow-arch-defs_SOURCE_DIR}/utils/vpr_io_place.py
             ${symbiflow-arch-defs_SOURCE_DIR}/utils/vpr_place_constraints.py
             ${symbiflow-arch-defs_SOURCE_DIR}/utils/eblif.py
-          DESTINATION bin/python
+          DESTINATION bin/python-utils
           PERMISSIONS WORLD_EXECUTE WORLD_READ OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE)
 
   install(FILES ${symbiflow-arch-defs_SOURCE_DIR}/utils/lib/parse_pcf.py
-          DESTINATION bin/python/lib)
+          DESTINATION bin/python-utils/lib)
 
 
   # install prjxray techmap
   install(DIRECTORY ${symbiflow-arch-defs_SOURCE_DIR}/xc/${FAMILY}/techmap
-          DESTINATION share/symbiflow/techmaps/${FAMILY}_vpr)
+          DESTINATION share/techmaps/${FAMILY}_vpr)
```

Comment:
Why did we remove this subdir?

Reply:
This change didn't get propagated to the scripts?

Reply:
This was a leftover, Fixed.